platform/viewer/package.json

```diff
@@ -24,7 +25,7 @@
     "build:viewer:package": "yarn run build:package",
     "dev": "cross-env NODE_ENV=development webpack-dev-server --config .webpack/webpack.pwa.js",
     "dev:orthanc": "cross-env NODE_ENV=development PROXY_TARGET=/dicom-web PROXY_DOMAIN=http://localhost:8042 APP_CONFIG=config/docker_nginx-orthanc.js webpack-dev-server --config .webpack/webpack.pwa.js",
-    "dev:viewer": "yarn run dev",
+    "dev:viewer": "yarn run dev --port 5001",
```

Comment:
Is it required in lerna 5.x to define port?

Reply:
Not needed. I added as part of my local test because of 3000 conflict. Removed it in new commit